README.md

```diff
@@ -1,68 +1,70 @@
+# Getting Started with Create React App
 
-![](https://img.shields.io/badge/Microverse-blueviolet)
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-# Bookstore
+## Available Scripts
 
->
+In the project directory, you can run:
 
-## Built With
+### `npm start`
 
-- React
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
```

Comment:
### 🛑 Required Change
- [ ] Kindly update the README using the Microverse README template and customize it reflect what you have done in this project milestone so far